wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/component/ChipScreenHeaders.kt

```diff
@@ -15,8 +15,21 @@ import androidx.wear.compose.material.Text
 @Composable
 fun ScreenHeaderChip(
     @StringRes text: Int,
+    modifier: Modifier = Modifier,
     textColor: Color? = null,
+) {
+    ScreenHeaderChip(
+        text = stringResource(text),
+        textColor = textColor,
+        modifier = modifier,
+    )
+}
+
+@Composable
+fun ScreenHeaderChip(
+    text: String,
```

Comment:
With the filter page I needed to pass the name of the filter which is entered by the user.